src/api_server/swagger/firecracker.yaml

```diff
@@ -62,11 +62,11 @@ paths:
 
   /boot-source:
     put:
-      summary: Creates or updates the boot source.
+      summary: Creates or updates the boot source. Pre-boot only.
       description:
         Creates new boot source if one does not already exist, otherwise updates it.
         Will fail if update is not possible.
-        Note that the only currently supported boot source is LocalImage.
+        It is only supported pre-boot.
```

Comment:
The summary already says it is pre-boot.

Reply:
The information is intentionally repeated in both the summary and the detailed description.

Do you think it's better to keep only one? In which case, which one?

Reply:
This is a constraint, it would make sense to keep it in the summary while the description provides addional details. I would mention it a second time only if I add details about it like the error returned if the API is called post boot.